releng/docker/image-build.sh

```diff
@@ -1,13 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 # vim: softtabstop=2 shiftwidth=2 expandtab
 
 usage() {
   cat <<-EOF
-	USAGE: $0 [-h] [-p <pkg>] <tag> [zbm-commit-like]
+	USAGE: $0 [-h] [-r <repo>] [-R <repo>] [-p <pkg>] <tag> [zbm-commit-like]
 	Create a container image for building ZFSBootMenu
 	
 	-h: Display this message and exit
 	
+	-r <repo>:
+	   Mount the specified host directory, which should be a valid XBPS
+	   repo, into the build container when installing packages
+	
+	-R <repo>:
+	   Configure XBPS to use the specified Void Linux package repository
+	
+	   The repo argument must be an absolute path (within the container),
+	   an HTTP or HTTPS URL, or "default" to specify the container default.
```

Comment:
don't think there's a need for this default workaround. any repo specified in the argv will take precedence over the one in the conf file, with the first-specified one having the highest priority. if you specify another mirror manually it will always take that over servercentral

Reply:
and similarly, `repository=` entries closer to the top of the list when `cat /etc/xbps.d/*.conf` will have priority

Reply:
The local repo is listed in a separate file that takes precedence, but the remote repos specified with `-R` are written separately to shadow the system default.

Previously, servercentral.com was the only supported repo, but the new logic allows that to be overridden with the builder's preferred remote. However, because users may prefer to augment rather than replace the default selection, the `default` magic word provides a more convenient way to reinstate that option (and even adjust the ordering of the remotes).